server/views/pages/appointments/create-and-edit/bulk-appointments/upload-bulk-appointment.njk

```diff
@@ -18,13 +20,39 @@
                     id: "file",
                     name: "file",
                     label: {
-                        text: "Upload a CSV file to create appointments",
+                        text: "Upload your list of prison numbers",
                         classes: "govuk-label--l",
                         isPageHeading: true
                     },
+
+                    hint: {
+                        text: "Your list of prison numbers must be in a CSV file. You can include appointment start
+                        and end times using the 24-hour clock. Fr example 14:00 not 2:00pm"
+                    },
                     errorMessage: validationErrors | findError('file')
                 }) }}
 
+                {{ govukAccordion({
```

Comment:
The design in the ticket uses a https://design-system.service.gov.uk/components/details/ component rather than an accordion. The accordion looks different and adds a "Show all sections" header. Needs changing to use details

Reply:
done